src/ontology/uberon-edit.obo

```diff
@@ -45,9 +45,6 @@ synonymtypedef: PENDING_REVIEW "expert consultation and attribution required"
 synonymtypedef: SENSU "taxonomic disambiguation"
 synonymtypedef: SYSTEMATIC "a systematic synonym used as the base name for design patterns"
 default-namespace: uberon
-idspace: EFO http://www.ebi.ac.uk/efo/EFO_ 
```

Comment:
can you check which version of Protege you are using? These should not disappear, it leads me to think you're using an older version with an older owlapi